python/hopsworks/core/environment_api.py

```diff
@@ -99,30 +110,37 @@ def get_environment(self):
 
         env_api = project.get_environment_api()
 
-        env = env_api.get_environment()
+        env = env_api.get_environment("my_custom_environment")
 
         ```
         # Returns
```

Comment:
`# Arguments` is missing with the description of the new paramenter.

Reply:
resolved